whisper-fine-tuning-sprint/README.md

```diff
@@ -0,0 +1,635 @@
+[comment]: # For more information on how to train an acoustic model on one of OVH's GPU - see [How to fine-tune a speech recognition model with OVHcould](#how-to-fine-tune-with-ovh-cloud).
+
+During the event, the speech recognition system will be evaluated on both the Common Voice `"test"` split 
+of the participants' chosen language as well as the *real-world* `"dev"` data provided by 
+the Hugging Face team. 
+At the end of the whisper fine-tuning sprint, the speech recognition system will also be evaluated on the
+*real-world* `"test"` data provided by the Hugging Face team. Each participant should add an 
+`eval.py` script to her/his model repository in a specific format that lets one easily 
+evaluate the speech recognition system on both Common Voice's `"test"` data as well as the *real-world* audio 
+data. Please read through the [Evaluation](#evaluation) section to make sure your evaluation script is in the correct format. Speech recognition systems
+with evaluation scripts in an incorrect format can sadly not be considered for the Challenge.
+
+At the end of the event, the best performing speech recognition system 
+will receive a prize 🏆 - more information regarding the prizes can be found under [Prizes](#prizes).
+
+We believe that framing the event as a competition is more fun, but at the core, the event is about
+creating speech recognition systems in as many languages as possible as a community.
+This can be achieved by working together, helping each other to solve bugs, share important findings, etc...🤗
+
+**Note**:
+Please, read through the section on [Communication & Problems](#communication-and-problems) to make sure you 
+know how to ask for help, etc...
+All important announcements will be made on discord. Please make sure that 
+you've joined [this discord channel](https://hf.co/join/discord)
+
+Also, please make sure that you have been added to the [Speech Event Organization](https://CREATE_AN_ORGANISATION_ON_HUB). 
+You should have received an invite by email. If you didn't receive an invite, please contact the organizers, *e.g.* Sanchit or VB directly on discord.
+
+[comment]: # VB: create an organisation on the hub.
+
+## Important dates
+
+![timeline](https://github.com/patrickvonplaten/scientific_images/raw/master/Robush%20Speech%20Challenge.png)
+
+[comment]: # VB - Create an infographic for the timeline.
+
+## Data and preprocessing
+
+In this section, we will quickly go over how to find suitable training data and 
+how to preprocess it.
+
+To begin with, **all data except Common Voice's `"test"` data can be used as training data.**
+The exception includes all Common Voice versions as the test data split of later Common Voice versions often
+overlaps with the one of previous versions, *e.g.* the test data of Common Voice 7 in English is 
+to a big part identical to the test data of Common Voice 6 in English:
+
+```python
+load_dataset("mozilla-foundation/common_voice_11_0", "en", split="test") 
```

Comment:
I think doing this requires previously going to the Hub and accepting the gate, so we should document that